pages/content/amp-dev/documentation/guides-and-tutorials/learn/validation-workflow/validation_errors.md

```diff
@@ -264,7 +282,9 @@ Detailed messages for this can be one of the following:
 * "Disallowed @supports in CSS"
 * "Disallowed @document in CSS"
 * "Disallowed @page in CSS"
+[filter formats="websites, stories, ads"]
```

Comment:
I think this filter should be the opposite: viewport is disallowed only for email

Reply:
Yeah, this filtering systems means it will only show on whats list, not hide